WordPress/WordPressTest/Dashboard/BlogDashboardServiceTests.swift

```diff
@@ -10,8 +10,7 @@ import Nimble
 ///
 /// We weren't able to figure out why but it seems a race condition + Core data
 /// For now, renaming the suite to change the execution order solves the issue.
-class ZBlogDashboardServiceTests: XCTestCase {
-    private var contextManager: ContextManagerMock!
+class ZBlogDashboardServiceTests: CoreDataTestCase {
```

Comment:
Once all your work is done, it might be worth to try removing this `Z`-prefix workaround and see if the issue is still there.

I'm dubious it will help much, because I have a feeling (a true feeling, not an informed suspicion, so I could be very wrong) the reason has more to do with how the production code interacts with Core Data rather than the tests. But, still worth a shot.

Reply:
I can definitely give it a go 👍

Reply:
Removed in https://github.com/wordpress-mobile/WordPress-iOS/pull/18660